db/schema.rb

```diff
@@ -10,11 +10,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_11_06_022510) do
+ActiveRecord::Schema.define(version: 2018_11_27_010432) do
```

Comment:
oops, I guess my migrations from when I was playing around over the break are still here...  I'll have to clean that up